resources/tests/scripts/demonstrate_require.lua

```diff
@@ -0,0 +1,11 @@
+local demo = require "lib/demo"
```

Comment:
The idea is that you ought to be able to include other files in your scripts directory. I only had ambitions for native lua files and haven't tested mixed lua/C setups. 

I restricted the path to _just_ scripts-directory, though it's still possible for the filter programmer to change the path to be global. That is, I am _assuming_ local includes but not denying global, if the user goes to the trouble of modifying the restricted path appropriately.